sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -620,7 +621,7 @@ static int get_sdw_dailink_info(const struct snd_soc_acpi_link_adr *links,
 		part_id = SDW_PART_ID(adr);
 		id = find_codec_info_part(part_id);
 		if (id < 0)
-			return -EINVAL;
+			return id;
```

Comment:
avoid 'id' and use something meaningful and consistent throughout this file.

Reply:
yes, I can fix it.  I was tired of 80-char limitation with long name and had to separate one line code to two lines, so I often use short name.